proto/vtctlservice.proto

```diff
@@ -35,4 +35,11 @@ service Vtctld {
   rpc GetKeyspace(vtctldata.GetKeyspaceRequest) returns (vtctldata.GetKeyspaceResponse) {};
   // GetKeyspaces returns the keyspace struct of all keyspaces in the topo.
   rpc GetKeyspaces(vtctldata.GetKeyspacesRequest) returns (vtctldata.GetKeyspacesResponse) {};
+  // InitShardPrimary sets the initial primary for a shard. Will make all other
+  // tablets in the shard replicas of the provided primary.
+  //
+  // WARNING: This could cause data loss on an already replicating shard.
+  // PlannedReparentShard or EmergencyReparentShard should be used in those
+  // cases instead.
+  rpc InitShardPrimary(vtctldata.InitShardPrimaryRequest) returns (vtctldata.InitShardPrimaryResponse) {};
```

Comment:
I'm genuinely unsure what the right API here is. Is all we want the stream of updates? Maybe this is a case where we actually want just a `stream logutil.Event` response 🤔

Reply:
I don't know either :) What you have is fine for now and we can fix forward if we decide otherwise.